x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/range_editor.tsx

```diff
@@ -14,14 +15,88 @@ import {
   EuiFlexGroup,
   EuiButtonIcon,
   EuiToolTip,
-  EuiIconTip,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiText,
+  EuiCode,
+  EuiIcon,
 } from '@elastic/eui';
-import { IFieldFormat } from 'src/plugins/data/public';
+import type { IFieldFormat } from 'src/plugins/data/public';
+import { UI_SETTINGS } from '../../../../../../../../src/plugins/data/public';
 import { RangeColumnParams, UpdateParamsFnType, MODES_TYPES } from './ranges';
 import { AdvancedRangeEditor } from './advanced_editor';
 import { TYPING_DEBOUNCE_TIME, MODES, MIN_HISTOGRAM_BARS } from './constants';
 import { useDebounceWithOptions } from '../helpers';
 
+const GranularityHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const granularityPopoverTitle = i18n.translate(
+    'xpack.lens.indexPattern.ranges.granularityPopoverTitle',
+    {
+      defaultMessage: 'How does interval granularity work?',
+    }
+  );
+  return (
+    <EuiPopover
+      ownFocus
+      isOpen={isPopoverOpen}
+      button={
+        <EuiButtonIcon
+          iconType="questionInCircle"
+          color="subdued"
+          onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+          aria-label={granularityPopoverTitle}
+        />
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      anchorPosition="leftCenter"
```

Comment:
The current position of the popover overlaps the label for the form. Would it be possible to change the anchorPosition, and also to make the popover button trigger from `EuiFormRow` instead of the ButtonIcon?

Reply:
Did a quick change to the `EuiFormRow` positioning:

<img width="408" alt="Screenshot 2020-12-23 at 18 07 48" src="https://user-images.githubusercontent.com/924948/103021067-d85f8580-4549-11eb-9993-ab7e69abdd94.png">

It looks a bit confusing to me given the `Create custom ranges` link just below.

Probably putting it on the right hand side may looks better: my only concern is that if you pass to custom intervals and then get back the cursor will hover it (not a major concern):

<img width="401" alt="Screenshot 2020-12-23 at 18 12 31" src="https://user-images.githubusercontent.com/924948/103021557-a1d63a80-454a-11eb-979c-aaff40fb7012.png">

In this last case it makes sense to move the anchorPosition to `upRight`.

Reply:
Pushed the second version in the PR.